action.yml

```diff
@@ -17,6 +17,14 @@ inputs:
 
       If not set a search will be performed to find a meticulous.json file in the current directory or the nearest parent directory.
     required: false
+  max-retries-on-failure:
```

Comment:
The corresponding argument should be added to `env` just below. This should fix the tests.

Reply:
Ooops yes!